.github/workflows/go.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v4
       with:
-        go-version: '1.21'
+        go-version-file: 'go.mod'
```

Comment:
Ah. Nice! Question: Does this look at the `go` or `toolchain` field?

Reply:
https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file